charmcraft/extensions/gunicorn.py

```diff
@@ -134,6 +134,17 @@ def _get_root_snippet(self) -> dict[str, Any]:
                     "description": f"{self.framework} application image.",
                 },
             },
+            "charm-libs": [
+                {"lib": "traefik_k8s.ingress", "version": "2"},
+                {"lib": "observability_libs.juju_topology", "version": "0"},
```

Comment:
Is there any worry with version 0 libs here that they'll make breaking changes?

Reply:
I think for some reason it is common to start the versions on version 0 for charm libraries, I have seen it a lot. There generally shouldn't be breaking changes within any major version

Reply:
Hopefully LIBPATCH will not introduce non-breaking changes. https://juju.is/docs/sdk/document-your-charm-library.

I do not think the LIBAPI 0 has the usual semantics of semver with major number 0 (that any breaking change can happen). In the wild, we haven't have problems with that either.